uwblueprint/project-read-frontend
# add MC options to form field selects

### Notion ticket link



[Add options for multiple choice questions](https://www.notion.so/uwblueprintexecs/Add-options-for-multiple-choice-questions-eb08b5beafaa4257b94344b661703c40)



### Implementation description

- Added options array to `Field` type, to match design in backend
- Display options on Registration Form



### Steps to test

1. Create dummy data
2. Click all the drop downs in the registration form, you should see options



### What should reviewers focus on?

- Should any test cases be added? Pls comment
- Are aria-labels applicable here?

### Checklist
